src/la/vector_mpi.hh

```diff
@@ -51,6 +52,12 @@ public:
     : communicator_(comm) {
         resize(local_size);
     }
+    
+    /// Create PETSc vector with ghost values whose indices are specified in @p ghost_idx.
+    VectorMPI(unsigned int local_size, std::vector<LongIdx> &ghost_idx)
+    : communicator_(PETSC_COMM_WORLD) {
+        resize(local_size, ghost_idx);
+    }
 
```

Comment:
Add support for scatters of ghost values, PETSc functions  VecGhostUpdateBegin  VecGhostUpdateEnd:
proposed methods:
ghost_update_begin(bool add = false)
ghost_update_end() .. should remember kind of scatter
... also some other methods should have asserts to no be called while update is in process

Reply:
Implemented in local_to_ghost_begin(), local_to_ghost_end(), ghost_to_local_begin(), ghost_to_local_end().